### ISSUE TITLE
Missings

### ISSUE THREAD

**Issue by cansirri** [strawpoll/strawpoll#37]
1.It will be great if you add an ending time of the polls. Time limited polls make people makes their choices early and show the result in time. It can be optional.

2.Other thing is I want to see how many people saw my poll page and how many of them vote in multiple choice polls (Not the quantity of the votes, quantity of the people).If I send my poll to 10 men, I need to be sure all of them voted or not. You should also add it to results page. If you do that I'll be greatful.

3.Creator should limit the quantity of multiple choice polls. For example; you are allowed to check at least 3 box in a 7 optioned poll.

Thanks

**Comment by summit2g** [comment]
If you still looking for these features, atleast featur 3 is available in another poll website streampoll.tv Also you can disable your poll so its not timed as with ur idea #1, but still.

**Comment by Chew** [comment]
These are all awesome ideas, I hope the dev sees this.

**Comment by dandirks** [comment]
I agree, these are very good ideas. I will certainly evaluate them all and get them on the roadmap.
